src/web/config.md

```diff
@@ -10,9 +10,9 @@ The latter allows you to plug in values that you would have defined here.
 -->
 
 <!-- META DEFINITIONS -->
-@def title        = "YourPackage.jl"
+@def title        = "{{ title }}"
```

Comment:
```suggestion
@def title        = "{{ title }}"
```
These should be in accordance with the general style, i.e. `{{title}}` instead of `{{ title }}`. Or this style should be implemented everywhere, since it is more readable?

Reply:
Sorry what do you mean with your change? You want to have `{{title}}` without spaces everywhere? With the change by @tlienart this would work yes. I somehow find the other more readable.

Reply:
Oh, I edited the suggested change. I don't have the strict preference, I just want it to be uniform across the project. I do agree `{{ title }}` is more readable, and that's also what Jekyll and Hugo use.

Reply:
I see you have the `space` syntax in `foot.html` but normally without spaces.

Reply:
Maybe you can use `{{ ... }}` everywhere if you want? :laughing: